src/routes/LoginPage/LoginPage.js

```diff
@@ -72,33 +147,132 @@ export default class LoginPage extends Component {
     }
   }
 
-  render() {
+  renderLogin() {
     return (
       <>
         <div className='loginWrapper'>
           {this.renderPageTitle()}
           {this.renderErrorMessage()}
-          <form onSubmit={this.state.shelter ? this.handleShelterSubmit : this.handleUserSubmit}>
+          <form onSubmit={this.state.shelter ? this.handleShelterSubmit : this.handleUserLoginSubmit}>
             <div className='form-group'>
               <label>
                 <p>Email Address</p>
-                <input type='email' className='form-control' onChange={e => {
+                <input type='email' className='form-control' value={this.state.email} onChange={e => {
                   this.setState({ email: e.target.value });
                 }} />
               </label>
             </div>
             <div className='form-group'>
               <label>
                 <p>Password</p>
-                <input type='password' className='form-control' required onChange={e => {
+                <input type='password' className='form-control' required value={this.state.password} onChange={e => {
                   this.setState({ password: e.target.value });
                 }} />
               </label>
             </div>
             <button className='btn' type='submit'>Submit</button>
           </form>
+          <button className='btn btn-link' onClick={this.handlePageSwitch}>Dont have an account? Create one!</button>
```

Comment:
My turn to nitpick lol Is there a reason for using `Dont` instead of `Don't`?

Reply:
Nope. Just a typo lol, good catch!